common/utils.go

```diff
@@ -58,6 +75,119 @@ func NewCanal(config *mysql.Config, tables []string) (*canal.Canal, error) {
 	return c, nil
 }
 
+// nopLogger disables logging for go-mysql/canal.
+type nopLogger struct{}
```

Comment:
This is a nit pick but why not use the a logger with a null handler? https://github.com/siddontang/go-log/blob/1e957dd83bed18c84716181da7b80d4af48eaefe/log/handler.go#L37-L54

Reply:
Because I did not see it :S. In my defense, I don't like to add a dependency to remove a bit of boilerplate, but `canal.Canal` also uses `go-log`. I guess we can afford it.

Reply:
Yea, I saw that.. the thousand go logging libraries out there. log/slog is doing a good job.